### PR TITLE
(maint) Add basic information to the README

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+* [DL4000](https://github.com/hadolint/hadolint/wiki/DL4000) - MAINTAINER is deprecated
+* [DL4001](https://github.com/hadolint/hadolint/wiki/DL4001) - Don't use both wget and curl
+
+### `puppet-docker pull`
+
+Pull the specified image, i.e. 'puppet/puppetserver'. *NOTE*: If you don't include the tag you want to pull, `puppet-docker pull` will pull all tags for that image.
+
+### `puppet-docker push`
+
+Push images build from the dockerfile in DIRECTORY to hub.docker.com. This task will fail if you do not have a version specified in your dockerfile. It will push both a versioned and a 'latest' tagged image.
```
Comment (mwaggett) [inline README.md]: 'build' -> 'built'

Comment (underscorgan) [reply README.md]: updated

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+  --dockerfile=<dockerfile>  File name for your dockerfile [default: Dockerfile]
+```
+
+### `puppet-docker build`
+
+Build a docker image based on the dockerfile in DIRECTORY.
+
+### `puppet-docker lint`
+
+Run [hadolint](https://github.com/hadolint/hadolint) on the dockerfile in DIRECTORY. The lint task runs on the `lukasmartinelli/hadolint` container with the following rule exclusions:
```
Comment (shrug) [inline README.md]: Still reference 'lukasmartinelli'

Comment (underscorgan) [reply README.md]: updated

**Pull request merged**